docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -0,0 +1,47 @@
+General configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table::
+  :widths: 30, 40, 30
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default value
+  * - ``case-insensitive-name-matching``
+    - Support case insensitive database and collection names
+    - False
+  * - ``case-insensitive-name-matching.cache-ttl``
+    -
+    - 1 minute
+  * - ``metadata.cache-ttl``
+    - Duration for which metadata, including table and column statistics, is
+      cached
+    - 0 (disabled caching)
+  * - ``metadata.cache-missing``
+    - Cache the fact that metadata, including table and column statistics, is
+      not available
+    - False
+  * - ``allow-drop-table``
+    - Allow connectors to drop tables
+    - False
+  * - ``join-pushdown.enabled``
+    - Enable join pushdown
+    - False
+  * - ``aggregation-pushdown.enabled``
+    - Enable aggregation pushdown
+    - True
+  * - ``topn-pushdown.enabled``
+    - Enable TopN pushdown
+    - True
+  * - ``write.batch-size``
+    - Maximum number of rows to insert in a single batch
+    - 1000
+  * - ``insert.non-transactional-insert.enabled``
+    - Controls how inserts are performed. By default, data insertion is
```

Comment:
Should be "Control"

Reply:
Removed from this PR. Will be a follow-up.